src/main/resources/db/migration/mysql/V1_3__init.sql

```diff
@@ -1,18 +1,19 @@
 CREATE TABLE `troop` (
   `id`           bigint(20) NOT NULL AUTO_INCREMENT,
+  `level`        bigint(20) UNSIGNED NOT NULL,
   `HP`           bigint(20) DEFAULT NULL,
   `attack`       bigint(20) DEFAULT NULL,
   `defense`      bigint(20) DEFAULT NULL,
-  `started_at`   bigint(20) DEFAULT NULL,
-  `finished_at`  bigint(20) DEFAULT NULL,
+  `started_at`   TIMESTAMP (6),
+  `finished_at`  TIMESTAMP (6),
```

Comment:
I would change TIMESTAMP to DATETIME. Basically they're the same, but DATETIME is used when you need both date and time information (src: https://stackoverflow.com/questions/409286/should-i-use-the-datetime-or-timestamp-data-type-in-mysql)

Reply:
https://www.eversql.com/mysql-datetime-vs-timestamp-column-types-which-one-i-should-use/ apparently it doesn't really matter which one you use. Based on performance, TIMESTAMP seems to be the better solution.

It might be an important information though that Hibernate automatically generates DATETIME.